plugin-tester-scala/build.gradle

```diff
@@ -5,55 +5,30 @@ buildscript {
   }
 
   dependencies {
-    // see https://plugins.gradle.org/plugin/com.lightbend.akka.grpc.gradle
-    // for the currently latest version.
-    //classpath 'gradle.plugin.com.lightbend.akka.grpc:akka-grpc-gradle-plugin:0.1'
-
-    // For testing on travis, we use another group id:
-    classpath 'com.lightbend.akka.grpc:akka-grpc-gradle-plugin:SNAPSHOT'
-
-    classpath "gradle.plugin.com.github.maiflai:gradle-scalatest:0.22"
+    classpath "com.lightbend.akka.grpc:akka-grpc-gradle-plugin:${project.getProperty("akkaGrpcTest.pluginVersion")}"
   }
 }
 
 plugins {
   id 'scala'
   id 'application'
-  id "com.github.maiflai.scalatest" version "0.22"
 }
 
 apply plugin: 'com.lightbend.akka.grpc.gradle'
 
-apply plugin: "com.github.maiflai.scalatest"
```

Comment:
Oh, the `maiflai.scalatst` plugin is not required? That's nice! @ennru and I were fighting it for a while in the  past.

Reply:
This plugin often brings a lot of weird errors related to classpath when running tests cin CI or from command line.
In our company we've abandoned it in favor of @RunWith annotations.